devops/yarn/README.md

```diff
@@ -0,0 +1,47 @@
+# Prerequisites
```

Comment:
We may need to change the format of help doc from md to rst, because other doc in Ray all rst format, and you could find the format change tools by Google.

Reply:
ok~